<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1457</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="counting">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Counting Cells</string>
   </property>
  </widget>
  <widget class="QLabel" name="OR_Image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="GY_Image">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="OR_Image_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ND_Image">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TH_Image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Original Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gray Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>850</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Threshold Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>850</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Morphology Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="Counting_layer">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>631</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>160</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Aplikasi Hitung Sel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>630</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>630</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>White Adipose Count</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>690</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Matriks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>670</y>
     <width>681</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>700</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>700</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>750</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_4">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>750</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_5">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>740</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_6">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>690</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_7">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>690</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_8">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>740</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>830</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>830</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pixel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>830</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>filename</zorder>
  <zorder>browse</zorder>
  <zorder>counting</zorder>
  <zorder>pushButton</zorder>
  <zorder>OR_Image</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>GY_Image</zorder>
  <zorder>OR_Image_3</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>ND_Image</zorder>
  <zorder>TH_Image</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>Counting_layer</zorder>
  <zorder>label</zorder>
  <zorder>filename_3</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>lcdNumber_2</zorder>
  <zorder>lcdNumber_3</zorder>
  <zorder>lcdNumber_4</zorder>
  <zorder>lcdNumber_5</zorder>
  <zorder>lcdNumber_6</zorder>
  <zorder>lcdNumber_7</zorder>
  <zorder>lcdNumber_8</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>lcdNumber_9</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
